tests/coalaHTMLTest.py

```diff
@@ -40,10 +54,14 @@ def test_output_file(self):
                           "-c", os.devnull,
                           "-b", "LineCountBear",
                           "-f", re.escape(filename),
-                          "--noupdate",
-                          "--nolaunch")
+                          "--noupdate")
 
             with open(self.result_file, 'r') as fp:
                 noupdate_file = fp.read()
 
         self.assertEqual(update_file, noupdate_file)
+        # We `chdir` before spawning the server
+        shutil.rmtree('../coalahtmlapp', ignore_errors=True)
```

Comment:
Either way, comment is not very informative.
Basically, all the tests in this class will benefit from this tearDown. :P
Also, I though that all the tests in this class, if there will be more, will be related to running coala_html, so they will require this cleanup.